app/assets/javascripts/components/pipelineList.jsx

```diff
@@ -0,0 +1,70 @@
+class PipelineList extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+  }
+
+  componentDidMount() {
+    $('.tooltipped').tooltip({delay: 50});
+  }
+
+  render() {
+    return (
+      <div>
+        <Header />
+        <div className="sub-header-home">
+          <div className="container">
+            <div className="content">
+              <div className="title">
+                All Projects  > { this.props.outputData.project_info.name }
+              </div>
+
+              <div className="sub-title">
+                { this.props.outputData.project_info.name }
+              </div>
+
+              <div className="title-filter">
+                <i className="fa fa-bar-chart" aria-hidden="true"></i>
+                <span>PIPELINES OUTPUTS</span>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="content-wrapper">
+          <div className="container sample-container">
+            {!this.props.pipelineOutputs ? 'Nothing to show' :
+              <table className="bordered highlight">
+                <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Date Uploaded</th>
+                  <th>Total Reads</th>
+                  <th>Final Reads</th>
+                  <th>Percentage Reads</th>
+                </tr>
+                </thead>
+
+                {this.props.pipelineOutputs.map((output, i) => {
```

Comment:
Rather than have all of this logic in the render function, I'd suggest add a renderPipelineOutputs function that generates all the tbody's